src/side-navigation/side-navigation.scss

```diff
@@ -100,20 +101,20 @@
       }
     }
 
-    > .iui-icon:not(.iui-user-icon) {
+    > .iui-button-icon:not(.iui-user-icon) {
```

Comment:
`:not(.iui-user-icon)` can be removed now (also search for other instances in the project)

Reply:
Wait I think tests are failing because of the removal, so maybe not.